src/CSET/operators/plot.py

```diff
@@ -971,43 +972,56 @@ def plot_line_series(
 
     Returns
     -------
-    Cube
-        The original cube (so further operations can be applied).
+    iris.cube | iris.cube.CubeList
+        An iris cube of the plotted data, or a cubelist of the
```

Comment:
```suggestion
        The original cube or CubeList (so further operations can be applied).
```
The plotting happens as a side effect here, so the returned data from this function hasn't changed.